solr/CHANGES.txt

```diff
@@ -65,6 +65,8 @@ Optimizations
 Bug Fixes
 ---------------------
 
+* SOLR-16477: Collection RENAME api creates broken alias
```

Comment:
nit: usually this goes at the bottom of the list.

Reply:
moved it to the bottom